src/schema/v2/notifications/Item/PartnerOfferCreatedNotificationItem.ts

```diff
@@ -0,0 +1,53 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import { connectionFromArray } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { artworkConnection } from "schema/v2/artwork"
+import { createPageCursors } from "schema/v2/fields/pagination"
+import { ResolverContext } from "types/graphql"
+
+export const PartnerOfferCreatedNotificationItemType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "PartnerOfferCreatedNotificationItem",
+  fields: {
+    artworksConnection: {
+      type: artworkConnection.connectionType,
+      args: pageable(),
+      resolve: async (
+        { object_ids },
+        args,
+        { artworksLoader, mePartnerOfferLoader }
+      ) => {
+        if (!mePartnerOfferLoader) return null
+        // object_ids should include a single partner offer id
+        if (object_ids.length === 0) return null
+
+        const { artwork_id } = await mePartnerOfferLoader(object_ids[0])
+
+        const { page, size } = convertConnectionArgsToGravityArgs(args)
+        const body = await artworksLoader({ ids: [artwork_id] })
+        const totalCount = body.length
+
+        return {
+          totalCount,
+          pageCursors: createPageCursors({ page, size }, totalCount),
+          ...connectionFromArray(body, args),
+        }
+      },
+    },
+    expiresAt: {
+      type: GraphQLString,
```

Comment:
We have a more structured type for timestamps, that support some common client formatting needs - https://github.com/artsy/metaphysics/blob/98e7b3b7f6dc00932f9614b87f885417d51a4a08/src/lib/date.ts - should likely use that.